developing.adoc

```diff
@@ -0,0 +1,45 @@
+
+
+= Developing {project}
+
+[[developing-overview]]
+== Overview
+
+The following sections describe how to build and test {project}.
+
+[[develop-prerequisites]]
+== Prerequisites
+
+- Git
+- Make
+- A recent Go distribution (>=1.11)
+
+[NOTE]
+====
+You should be able to develop the project on Linux, Windows or macOS.
+====
+
+[[set-up-dev-env]]
+== Setting Up the Development Environment
```

Comment:
I think we should be using sentence case for titles -- it's much simpler to use and understand.

```suggestion
== Setting up the development environment
```